base/applications/taskmgr/perfdata.c

```diff
@@ -144,16 +145,16 @@ CachedGetUserFromSid(
         if (EqualSid((PSID)&pEntry->Data, pSid))
         {
             wcsncpy(pUserName, pEntry->pszName, cwcUserName);
-            *pcwcUserName = cwcUserName;
+            *pcwcUserName = wcslen(pUserName);
```

Comment:
As this out value is (currently) unused, a simplification would be to change this parameter to an in-only one.
What do you think?

Reply:
Let's not introduce additional changes unrelated to the bug you wanted to fix.

Reply:
> additional changes unrelated to the bug

That would be a different/final fix/optimization, not an additional change.
That seems very related to this code...